src/main/java/fi/fmi/avi/model/swx/SpaceWeatherPhenomenon.java

```diff
@@ -0,0 +1,61 @@
+package fi.fmi.avi.model.swx;
+
+public interface SpaceWeatherPhenomenon {
```

Comment:
The more I think of this, the more I like the idea of this interface (and corresponding `SpaceWeatherPhenomenonImpl.java`) being an enum type, having values specified in Notation column of WMO [Register: Space Wx Phenomena](https://codes.wmo.int/49-2/SpaceWxPhenomena).

But it should work like this too, and is not a priority to change it.

Reply:
Actually, current implementation seem to complicate the JSON serialized format. See change in [spacewx-A2-3.json](https://github.com/spatineo/fmi-avi-messageconverter-iwxxm/commit/db0811e96cfb5a8d39b4520421a3c2c9de344bbf#diff-8c0fd5c7e3ecbf5229b092856b8f3fb6) for an example. An enum would result in more compact and clean JSON content, like the initial state of the example above.

Reply:
This is now covered by issue #55.